src/GeneralStateTestsFiller/VMTests/vmTests/push0GasFiller.yml

```diff
@@ -0,0 +1,66 @@
+# EIP-3855: PUSH0 instruction - test for gas price
+push0Gas:
+  env:
+    currentCoinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    currentDifficulty: '0x020000'
+    currentGasLimit: '89128960'
+    currentBaseFee: '10'
+    currentNumber: '1'
+    currentTimestamp: '1000'
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+
+  pre:
+    a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: 10000000
+      code: ''
+      nonce: '0'
+      storage: {}
+
+    0000000000000000000000000000000000000100:
+      balance: 0
+      # sstore(0, gas) push0 sstore(0, (sub(sload(0), gas))) stop
+      code: ':raw 0x5a6000555f5a6000540360015500'
```

Comment:
raw?

Reply:
I don't think PUSH0 is available in Solidity, that's why.

Reply:
can use custom lllc